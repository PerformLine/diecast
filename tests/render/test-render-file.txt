type={{ underscore $.type }}
{{ range $item := $.items -}}
- {{ $item.id }}
{{ end -}}
debug=true