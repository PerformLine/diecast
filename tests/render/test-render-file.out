type=other
- 1
- 3
- 5
debug=true